PowerEditor/src/ScitillaComponent/FindReplaceDlg.h

```diff
@@ -410,15 +411,43 @@ class Progress
 	Progress(HINSTANCE hInst);
 	~Progress();
 
-	HWND open(HWND hOwner = NULL, const TCHAR* header = NULL);
-	bool isCancelled() const;
-	void setPercent(unsigned percent, const TCHAR *fileName) const;
-	void setInfo(const TCHAR *info) const {
+	HWND open(HWND hCallerWnd = NULL, const TCHAR* header = NULL);
+	void close();
+
+	bool isCancelled() const

```

Comment:
> and thus little things can greatly impact compilation time.

I understand but I'm usually more worried about execution time since the program is going to be used considerably more than it will be compiled. Although here one function call more is not a big deal.